test/components/viral/system/time_ago_component_test.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'application_system_test_case'
+
+module System
+  class TimeAgoComponentTest < ApplicationSystemTestCase
+    test 'default' do
+      visit('/rails/view_components/viral_time_ago_component/default')
```

Comment:
**nitpick (blocking):** 
Can you make this into 2 tests:
1. The `tooltip` test is good (rename to `default`), but start it by making sure the `15 days ago` is there, then go into testing the tooltip.
2. Make a second test `including current time` to test when the user wants to pass in a custom current time (although I still don't know when that would ever occur).  Then test the same way you did the first one including the value and tooltip.

Reply:
Fixed in ebaa3e31e69d1e06fdfd0d9fcc6b28975e70ae59